tests/integration/templates/ssm_maintenance_window.yml

```diff
@@ -0,0 +1,43 @@
+Resources:
```

Comment:
This file isn't valid against AWS.
`MaintenanceWindowTask` is missing the window id and it seems the way the targets are specified on `MaintenanceWindowTarget` isn't compatible with how AWS expects them.